src/main/kotlin/lotto/model/GenerateRandomInt.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.model
+
+/**
+ * 랜덤 숫자 반환하는 클래스
+ * */
+object GenerateRandomInt {
+    private const val NUMBER_START = 0
+    private const val NUMBER_END = 6
+
+    fun generateNumbers(range: List<Int>): List<Int> =
+        (range).shuffled()
+            .subList(NUMBER_START, NUMBER_END)
```

Comment:
```suggestion
        (range)
            .shuffled()
            .subList(NUMBER_START, NUMBER_END)
```
함수 체이닝 컨벤션을 적용하면 좋을 것 같아요

ref. https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls